cpp_linter/cli.py

```diff
@@ -358,6 +362,26 @@ class Args(UserDict):
     help="""Set to ``true`` to prevent Pull Request
 reviews from requesting or approving changes.""",
 )
+_parser_args[("-C", "--delete-review-comments")] = dict(
+    default="true",
```

Comment:
## Don't delete on default
```suggestion
    default="false",
```
I'd prefer to leave the more destructive operations as opt-in. My main concern here is any replies to the initial/deleted review comment(s) would become harder to understand.